LibreNMS/Data/Store/InfluxDB.php

```diff
@@ -39,20 +39,25 @@ class InfluxDB extends BaseDatastore
     private $batchPoints = []; // Store points before writing
     private $batchSize = 0; // Number of points to write at once
     private $measurements = []; // List of measurements to write
+    private $isUdp = false; // Is this using a UDP transport
 
     public function __construct(private Database $connection)
     {
         parent::__construct();
         $this->batchSize = LibrenmsConfig::get('influxdb.batch_size', 0);
         $this->measurements = LibrenmsConfig::get('influxdb.measurements', []);
+        $this->isUdp = LibrenmsConfig::get('influxdb.transport', 'http') === 'udp';
 
         // if the database doesn't exist, create it.
-        try {
-            if (! $this->connection->exists()) {
-                $this->connection->create();
+        // this only works if the transport is TCP-based.
```

Comment:
Why wouldn't this work with UDP transport?  I know often the influxdb user does not have permissions to create a database.

Reply:
To be fair, the create will work.  It's the `exists()` that fails.  It raises the exception that the transport does not support querying.